cmd/postgres-operator/main.go

```diff
@@ -73,6 +74,8 @@ func initManager() (runtime.Options, error) {
 	options := runtime.Options{}
 	options.Cache.SyncPeriod = initialize.Pointer(time.Hour)
 
+	options.HealthProbeBindAddress = ":8081"
```

Comment:
👍 I see this is the default used by KubeBuilder as well.

(Though I see they also allow this to be configured via command CLI flag 🤔).

Reply:
Do we want to allow customers to set it on the Deployment? Or are we good with hardcoding it?